Buildscripts/DevicetreeCompiler/source/generator.py

```
@@ -97,50 +105,55 @@ def write_config(file, device: Device, bindings: list[Binding], type_name: str):
         file.write(f"{config_params_joined}\n")
     file.write("};\n\n")
 
-def write_device_structs(file, device: Device, parent_device: Device, bindings: list[Binding], verbose: bool):
+def write_device_structs(file, device: Device, parent_device: Device, bindings: list[Binding], devices: list[Device], verbose: bool):
```

Ruff (0.14.14)

[error] `Device` may be undefined, or defined from star imports

(F405)